dbmigrator/cli.py

```diff
@@ -28,18 +28,27 @@ def main(argv=sys.argv[1:]):
     parser.add_argument('--db-connection-string',
                         help='a psycopg2 db connection string')
 
+    parser.add_argument(
+        '--context',
+        help='Name of the python package containing the migrations')
+
     subparsers = parser.add_subparsers(help='commands')
     commands.load_cli(subparsers)
 
     args = parser.parse_args(argv)
     args = vars(args)
 
-    if args.get('config') and os.path.exists(args['config']):
+    if args.get('config'):
+        if not os.path.exists(args['config']):
+            raise Exception('config file not found')
         utils.get_settings_from_config(args['config'], [
             'migrations-directory',
             'db-connection-string',
             ], args)
-    utils.get_settings_from_entry_points(args)
+
+    if not args.get('context'):
+        args['context'] = os.path.split(os.path.abspath(os.path.curdir))[-1]
```

Comment:
Assuming `os.path.curdir` is `/home/karen/cnx-archive`, `os.path.dirname` returns `/home/karen` :disappointed: